_maps/map_files/KiloStation/KiloStation.dmm

```diff
@@ -12704,7 +12704,7 @@
 	height = 22;
 	id = "whiteship_home";
 	name = "SS13: Auxiliary Dock, Station-Port";
-	roundstart_template = /datum/map_template/shuttle/mining_common/kilo;
+	roundstart_template = null;
```

Comment:
unrelated change?

Reply:
person who made the map told me it was an incorrect var and it was causing issues some people have noticed (logically, theres a shuttle docked where the white ships supposed to dock). i said i would just add that in since im doing a map edit anyway, so not *completely* unrelated. i can open up a whole PR just for that one line though if need be

Reply:
also it already takes null from the parent doesnt it

Reply:
i have zero clue, but null is what that var is by default, and is also what it was given when i hit the var reset button.

i moved it to its own 1 line PR